libs/pageserver_api/Cargo.toml

```diff
@@ -17,5 +17,10 @@ postgres_ffi.workspace = true
 enum-map.workspace = true
 strum.workspace = true
 strum_macros.workspace = true
+url.workspace = true
```

Comment:
is this one used?

Reply:
Will be in subsequent changes to shard.rs, but not now.  Removed.